test/suites/websocket.ts

```diff
@@ -244,7 +244,8 @@ export default (context: TestContext) => {
         expect(message.data.events).toHaveLength(1);
         expect(message.data.events[0]).toEqual(mockMintWebSocketMessage);
         return true;
-      });
+      })
+      .close();;
```

Comment:
cosmetic, but we ended up with a bunch of extra semicolons

Reply:
Fixed. Thanks.